djstripe/models.py

```diff
@@ -547,6 +557,24 @@ def _attach_objects_post_save_hook(self, cls, data):
         # RelatedManager, so this must be done as part of the post save hook.
         cls._stripe_object_to_invoice_items(target_cls=InvoiceItem, data=data, invoice=self)
 
+    @classmethod
+    def upcoming(cls, **kwargs):
+        """ Gets the upcoming preview invoice (singular) for a customer.
+
+        See the documentation in ``djstripe.stripe_objects.StripeInvoice.upcoming()` for details on method parameters.
+        """
```

Comment:
Go ahead and remove this docstring, as it will automatically inherit the docstring from StripeInvoice.upcoming()